features/breed_list/src/main/java/com/example/catbreeds/breed_list/BreedListScreen.kt

```diff
@@ -53,6 +45,11 @@ fun BreedListScreen(
     val breeds by viewModel.breeds
     val searchQuery by viewModel.searchQuery
     val filteredBreeds by viewModel.filteredBreeds
+    var isSearchActive by remember { mutableStateOf(false) }
```

Comment:
On our project doing like this is the standard: 
```suggestion
    val isSearchActive = remember { mutableStateOf(false) }
```
Your way is not wrong by any means. Its just a heads up for when you move to our project. You can see the discussion [here](https://swordhealth.slack.com/archives/C02V2U0BDJ5/p1688047888168399)

Reply:
Thanks for the heads up, updated a few similar situations and will keep that in mind